test/integration-tests/Seaport.test.ts

```diff
@@ -141,4 +157,187 @@ describe('Seaport', () => {
       router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value: seaportValue })
     ).to.be.revertedWith('ExecutionFailed(0, "0x8baa579f")')
   })
+
+  describe('Seaport SELL 721', async () => {
+    let tubbyCats: ERC721
+    let weth: ERC20
+
+    beforeEach(async () => {
+      await resetFork(16385143) // txn is block 16385144
+      await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [ALICE_ADDRESS],
+      })
+      alice = await ethers.getSigner(ALICE_ADDRESS)
+      permit2 = (await deployPermit2()).connect(alice) as Permit2
+      router = (await deployUniversalRouter(permit2)).connect(alice) as UniversalRouter
+      planner = new RoutePlanner()
+      tubbyCats = new ethers.Contract(TUBBY_ADDRESS, ERC721_ABI) as ERC721
+      weth = new ethers.Contract(WETH.address, ERC20_ABI) as ERC20
+
+      // send 1 eth from alice to router.address for weth approval
+      await alice.sendTransaction({ to: router.address, value: ethers.utils.parseEther('1.0') })
+      expect(await ethers.provider.getBalance(router.address)).to.eq(ethers.utils.parseEther('1.0'))
```

Comment:
are there other instances of expect statements in beforeEach's in this repo??? Feels redundant... 
if it's bc there's no error if this reverts, could try 
```
(await alice.sendTransaction({ to: router.address, value: ethers.utils.parseEther('1.0') })).wait()
```

Reply:
Sg, I'll use the .wait more to get errors to throw